README.md

```diff
@@ -7,7 +7,7 @@ Building the site requires [Jekyll](http://jekyllrb.com/docs)
 The easiest way to install the right version of these tools is using
 [Bundler](https://bundler.io/) and running `bundle install` in this directory.
 
-See also https://github.com/apache/spark/blob/master/docs/README.md
+See also [https://github.com/apache/spark/blob/master/docs/README.md](https://github.com/apache/spark/blob/master/docs/README.md)
 
```

Comment:
Although `README.md` does not generate a corresponding `README.html` page in the `spark website project`, 
I believe that for consistency, the following statement still needs to be modified.